buildbot/buildbot/worker.py

```diff
@@ -110,7 +118,7 @@ def work(db, old_state, temp_state, done_state_or_func):
     """
     done_func = None
     if isinstance(done_state_or_func, str):
-        done_func = lambda _: done_state_or_func
+        done_func = lambda _: done_state_or_func  # noqa
```

Comment:
What's `noqa`?

Reply:
That's an "ignore this line" directive for the Python style checker I use (and recommend!), [flake8](https://flake8.pycqa.org/en/latest/). The default settings warn on `foo = lambda: ...`, instead wanting you to write the equivalent but more verbose `def foo(): ...`.